### PR TITLE
Add pet care android sample to sample-is

### PR DESCRIPTION
**Pull request by Achintha444**
## Purpose
[improvement] move pet care android application to sample-is

### CODE REVIEW

File: petcare-sample/android-mobile-application/README.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+ Java version >= 17
+ ```
+ ```
+ Android Studio version >= Giraffe
+ ```
+
+#### 1. Setup steps Identity Server
+1.1 Setup the latest version of WSO2 Identity Sever.
+
+1.2 Create a mobile application (Un tick the PKCE `Mandayory` checkbox)
```
Comment (pavinduLakshan) [inline petcare-sample/android-mobile-application/README.md:16]: Suggestion: Linking doc references to these steps would make it more convenient to a first time user to follow the readme.

File: petcare-sample/android-mobile-application/README.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+- Save the JSON in secure place (We need this for Android Attestation Credentials for application metadata)
+
+6.3 After that, Update Application Advanced properties.
+The application you created requires 2 properties to perform android attestation.
+- Android package name
+- androidAttestationServiceCredentials
+  - The JSON secret of Service Account downloaded. Note that this attribute is defined as a JSON object hence use the JSON key as it is.
+
+6.4 Go to the Google play console and create a new application for this application, after that you need to associate your cloud project for the app integrity. To do this,
+- Navigate to Release > App integrity. Under Play Integrity API select Link a Cloud project.
```
Comment (pavinduLakshan) [inline petcare-sample/android-mobile-application/README.md:248]:
```suggestion
- Navigate to Release > App integrity. Under Play Integrity API, select **Link a Cloud project**.
```

File: petcare-sample/android-mobile-application/README.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+- Choose the Cloud project you want to link to your app and this will enable Play Integrity API responses.
+- You can now integrate the Play Integrity API into your app.
+- Create a new release.
+  - For this we need a signed app bundle. To create a signed app bundle,
+    - In the android studio go to Build > Generate Signed App Bundle, and select app bundle.
```
Comment (pavinduLakshan) [inline petcare-sample/android-mobile-application/README.md:253]:
```suggestion
    - In the Android Studio, go to **Build** > **Generate Signed App Bundle**, and select app bundle.
```

File: petcare-sample/android-mobile-application/app/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (pavinduLakshan) [inline petcare-sample/android-mobile-application/app/.gitignore:1]: S. all we add a newline here?  

Check other places as well.

File: petcare-sample/android-mobile-application/app/release/app-release.aab
Comment (pavinduLakshan) [inline petcare-sample/android-mobile-application/app/release/app-release.aab:1]: Do we need to check this binary into version control?

**Comment by brionmario** [comment]
Shall we reconsider the folder structure?

```
petcare-sample/
    b2c/
       web-app
        mobile-app
              # Goes here?
```

**Comment by AnuradhaSK** [comment]
> Shall we reconsider the folder structure?
> 
> @AnuradhaSK any thoughts?
> 
> ```
> petcare-sample/
>     b2c/
>        web-app
>         mobile-app
>               # Goes here?
> ```

Yes this is the agreed folder structure (Note: a random name is given as mobile app name in this folder structure)

Screenshot 2024-03-21 at 14 31 57

File: petcare-sample/android-mobile-application/app/release/app-release.aab
Comment (Achintha444) [reply petcare-sample/android-mobile-application/app/release/app-release.aab:1]: Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d

File: petcare-sample/android-mobile-application/app/.gitignore (hunk 4 above)
Comment (Achintha444) [reply petcare-sample/android-mobile-application/app/.gitignore]: Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d

File: petcare-sample/android-mobile-application/README.md (hunk 3 above)
Comment (Achintha444) [reply petcare-sample/android-mobile-application/README.md]: Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d

File: petcare-sample/android-mobile-application/README.md (hunk 2 above)
Comment (Achintha444) [reply petcare-sample/android-mobile-application/README.md]: Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d

File: petcare-sample/android-mobile-application/README.md (hunk 1 above)
Comment (Achintha444) [reply petcare-sample/android-mobile-application/README.md]: Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d

**Review by brionmario — APPROVED** [review]

File: petcare-sample/b2c/mobile-app/petcare/README.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+#### 3. Setup Google Login
+3.1 Go to `https://console.cloud.google.com/` and create a new project.
+
+3.2 In the `credentials` section create two `Oauth Client IDs` one for `android app(Select Android)` and one for `WSO2 identity server(Select Web Application)`.
+> Follow the steps to get the SHA-1 key to create Oauth Client ID for our Android app
+> https://stackoverflow.com/a/67983215/10601286.
+> 
+> After that make sure to sync the gradle of the project. This can be done using the `Sync project with Gradle files` icon in the top right hand corner of the IDE (or in Apple `Shift + Command + O`).
+> <img width="1797" alt="Screenshot 2023-12-06 at 10 02 18" src="https://github.com/Achintha444/api-auth-android/assets/46097917/edc40aeb-395d-42c9-ae78-8a0499278c69">
+
```
Comment (pavinduLakshan) [inline petcare-sample/b2c/mobile-app/petcare/README.md:55]: Shall we update this URL to take the image from the samples-is repo itself?

Let's check and fix the other places as well.

Comment (Achintha444) [reply petcare-sample/b2c/mobile-app/petcare/README.md:55]: Fixed with https://github.com/wso2/samples-is/pull/445

**Pull request merged**